cmd/ssh.go

```diff
@@ -227,7 +231,7 @@ func createTempContextFile(execID string) (*os.File, error) {
 	return tmpFile, nil
 }
 
-func copySourceSCP(srcPath, tmpDstPath, dstPath string, connectionInfo types.ConnectionInfo, privKeyPath string) error {
+func copySourceSCP(fromTo []string, privKeyPath string) error {
```

Comment:
I moved this around because we can re-use this method if we just move the host/destination path up to the call site.